File: js/PROBLEMS/1118.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')
i=0
cont=0

while (1){
    x = parseFloat(lines.shift())

    if(x>=0 && x<= 10){
        cont+=x
        i++;
    }else{
        console.log('nota invalida')
    }
    if (i==2){
        console.log('media = ' + (cont/2).toFixed(2))
        while (1){
            console.log("novo calculo (1-sim 2-nao)");
            cal = lines.shift()
            if (cal == 1){
                i=0;
                cont=0;
                break;
            }else if (cal==2){
                return 0;
            }
        }
    }
}

File: python/PROBLEMS/1118.py
cont, i = 0, 0
while True:
    x = float(input())
    if x>=0 and x<=10:
        cont+=x
        i+=1
    else:
        print('nota invalida')
    if i==2:
        print(f'media = {(cont/2):.2f}')
        while (True):
            print('novo calculo (1-sim 2-nao)')
            cal = float(input())
            if cal == 1:
                i=0
                cont=0
                break;
            elif cal == 2:
                import sys
                sys.exit()